lib/URLShortener/URLShortener.php

```diff
@@ -130,6 +136,38 @@ private function _create_new_shortlink()
 			return $output;
 		}
 	}
+	
+	/**
+	 * Deletes a shortlink if the parameters are good
+	 * Added by @fellwell5
+	 *
+	 * @return void
+	 **/
+	private function _delete_new_shortlink()
```

Comment:
Would you mind changing this method name to `_delete_shortlink()` and update calls to it accordingly? I think that just makes a bit more sense.